### ISSUE TITLE
Site Tree Refresh

### ISSUE THREAD

**Issue by bossloper** [Nicola1971/BSxFLAT#2]
Not sure if it is my setup or still beta work in progress(?) but the site tree is not refreshing correctly for me.
I have copied over your new frames folder to replace my default frames folder.

If I add a new page it does not trigger a tree refresh. Also the 'Refresh site tree' and 'Purge' buttons do not work.

Looking at the code there are no mootools javascript libaries in 1.php
Even with those added some of the calls are now incorrect as they reference 'top.mainMenu.$' whereas now it is only 'top.$' as the javascript functions have moved out of mainMenu.

I'm not sure where to find all the calls to suggest how to fix this.

**Comment by Nicola1971** [comment]
Yes, "refresh cache" is the biggest issue with the menu moved inside 1.php 
The mainframe is the only frame refreshed :(
